### ISSUE TITLE
adb screenrecord process can't be stopped on device

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/appium#244]
In some scenarios we see that screenrecord continue to record videos and occupy space on device. As result it occupy all space too fast.
As a hotfix we have to restore agai handler for below log message:
```
root@456174331aa9:/home/androidusr# cat /tmp/log/appium.log | grep "c826141c-3d34-4ff0-87db-9f65cad80ea2"
2023-09-04 14:12:37:309 [AndroidUiautomator2Driver@9897 (c826141c)] [AndroidUiautomator2Driver@9897 (c826141c)] Session created with session id: c826141c-3d34-4ff0-87db-9f65cad80ea2
2023-09-04 14:13:27:265 [AppiumDriver@64d4] [AppiumDriver@64d4] New AndroidUiautomator2Driver session created successfully, session c826141c-3d34-4ff0-87db-9f65cad80ea2 added to master session list
2023-09-04 14:13:27:308 [AndroidUiautomator2Driver@9897 (c826141c)] [AndroidUiautomator2Driver@9897 (c826141c)] Cached the protocol value 'W3C' for the new session c826141c-3d34-4ff0-87db-9f65cad80ea2
2023-09-04 14:16:27:404 [AppiumDriver@64d4] [AppiumDriver@64d4] Removing session 'c826141c-3d34-4ff0-87db-9f65cad80ea2' from our master session list
```

it guarantees we stop recording during negative scenario when appium don't send delete request.

more stable solutions are under investigation in scope of #235  and #195

**Comment by vdelendik** [comment]
improvement is available in `appium:2.0.4-beta2`. @dhreben, I'm going to test it on my own as a hotfix for problematic device

**Comment by vdelendik** [comment]
everything is good, no complains
